scripts/instance/mask_rcnn/train_mask_rcnn.py

```diff
@@ -258,7 +271,33 @@ def validate(net, val_data, ctx, eval_metric, args):
                     full_masks.append(gdata.transforms.mask.fill(mask, bbox, (im_width, im_height)))
                 full_masks = np.array(full_masks)
                 eval_metric.update(det_bbox, det_id, det_score, full_masks)
-    return eval_metric.get()
+    if args.horovod and MPI is not None:
+        comm = MPI.COMM_WORLD
+        res = comm.gather(eval_metric.get_result(), root=0)
```

Comment:
why use `get_result` instead of conventional `get` for metrics?

Reply:
this is actually different, since get_result() return the internal list instead of returning the result, will change to a better name